packages/config/src/ios/DeviceFamily.ts

```diff
@@ -40,7 +33,7 @@ export function getDeviceFamilies(config: ExpoConfig): number[] {
  * @param deviceFamilies
  */
 export function formatDeviceFamilies(deviceFamilies: number[]): string | number {
-  return deviceFamilies.length === 1 ? deviceFamilies[0] : `"${deviceFamilies.join(',')}"`;
+  return `"${deviceFamilies.join(',')}"`;
```

Comment:
Change the return type of this function to just `string` so we don't add back the `number` functionality on accident